README.md

```diff
@@ -1,4 +1,4 @@
-# The half-yearly Nix Community Developer Workshop and Hackathon
+# The half-yearly Nix community developer workshop and NixOS release hackathon
```

Comment:
```suggestion
# The bi-annual[^1] Nix community developer workshop and NixOS release hackathon

[^1]: /baɪˈæn.ju.əl/ *adjective* happening twice a year
```

Reply:
It's actually ambiguous that way, it could as well be read as "every two years". Half-yearly sounds weird but at least leaves no room for misunderstanding.